containerfile/oc-client

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/appuio/oc:v4.16
+
+RUN python3.9 -m venv ~/.ansible
+RUN ~/.ansible/bin/pip install ansible-core kubernetes
+RUN ~/.ansible/bin/ansible-galaxy collection install kubernetes.core
```

Comment:
I think it would be worth adding `requirements.yml` (galaxy) and `requirements.txt` (pip) so that this can be reproduced outside of the container, and allow us to target either specific versions, or version range.

For example, we should mount a `requirements.yml` and run `ansible-galaxy install -r requirements.yml` instead of directly installing without version targets.

Then I would suggest writing docs on how to setup this environment without the containerized instance so that we have both methods available.